client/wordpress-com.js

```diff
@@ -314,54 +314,44 @@ if ( config.isEnabled( 'reader' ) ) {
 	} );
 }
 
-if ( config.isEnabled( 'vip' ) ) {
-	sections.push( {
-		name: 'vip',
-		paths: [ '/vip', '/vip/deploys', '/vip/billing', '/vip/support', '/vip/backups', '/vip/logs' ],
-		module: 'vip',
-		secondary: true
-	} );
-}
+sections.push( {
+	name: 'vip',
+	paths: [ '/vip', '/vip/deploys', '/vip/billing', '/vip/support', '/vip/backups', '/vip/logs' ],
+	module: 'vip',
+	secondary: true
+} );
 
-if ( config.isEnabled( 'help' ) ) {
-	sections.push( {
-		name: 'help',
-		paths: [ '/help' ],
-		module: 'me/help',
-		secondary: true,
-		enableLoggedOut: true,
-		group: 'me'
-	} );
-}
+sections.push( {
+	name: 'help',
+	paths: [ '/help' ],
+	module: 'me/help',
+	secondary: true,
+	enableLoggedOut: true,
+	group: 'me'
+} );
 
-if ( config.isEnabled( 'oauth' ) ) {
-	sections.push( {
-		name: 'auth',
-		paths: [ '/login', '/authorize', '/api/oauth/token' ],
-		module: 'auth',
-		secondary: false,
-		enableLoggedOut: true
-	} );
-}
+sections.push( {
+	name: 'auth',
+	paths: [ '/login', '/authorize', '/api/oauth/token' ],
+	module: 'auth',
+	secondary: false,
+	enableLoggedOut: true
+} );
 
-if ( config.isEnabled( 'manage/custom-post-types' ) ) {
```

Comment:
Check already present in https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/types/index.js#L9.